tsconfig.base.json

```diff
@@ -2,7 +2,6 @@
   "compilerOptions": {
     "target": "ESNext",
     "esModuleInterop": true,
-    "importsNotUsedAsValues": "error",
```

Comment:
Relying on eslint instead, or is there a different reason for the change?

Reply:
yep: it's deprecated now in TS 5

Reply:
👍 

Just catching up on the release notes...  [verbatimModuleSyntax](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#verbatimmodulesyntax) is nice for friends of ESM. This will ruffle some feathers in various bundlers.